### ISSUE TITLE
[NewPlugin] Njord BNPyc 1.0.1

### ISSUE THREAD

**Issue by Njord0** [Vector35/community-plugins#214]
Please add my plugin to the repository.

Yes, I've read the instructions, especially the part about creating an [actual release](https://binary.ninja/2019/07/04/plugin-manager-2.0.html#5-create-a-release) and creating the tag. 

I either used the UI, or the `hub` command to create a _real_ release.

Author: Me
Repo name: [BNpyc](https://github.com/Njord0/BNPyc)
Release: [1.0.1](https://github.com/Njord0/BNPyc/releases/tag/1.0.1)

**Comment by galenbwill** [comment]
I would suggest adding a screenshot that shows some (or all) ILs alongside your disassembly.

**Comment by Njord0** [comment]
> I would suggest adding a screenshot that shows some (or all) ILs alongside your disassembly.

I will but IL lifting is not implemented, there isn't much to show yet.

**Comment by galenbwill** [comment]
>> I would suggest adding a screenshot that shows some (or all) ILs alongside your disassembly.
>
> I will but IL lifting is not implemented, there isn't much to show yet.

Ok, that's fair. In that case, never mind the screenshot for now, but it's probably worth pointing out that lifting is not implemented yet, preferably towards the top of the readme.

**Comment by Njord0** [comment]
> > > I would suggest adding a screenshot that shows some (or all) ILs alongside your disassembly.
> > 
> > 
> > I will but IL lifting is not implemented, there isn't much to show yet.
> 
> Ok, that's fair. In that case, never mind the screenshot for now, but it's probably worth pointing out that lifting is not implemented yet, preferably towards the top of the readme.

You're right, I did create a new [release](https://github.com/Njord0/BNPyc/releases/tag/1.0.1) with the edited readme. I also edited my initial comment.

**Comment by psifertex** [comment]
Thanks for the submission, plugin has been added. 

Really curious to see how it works with some lifting, definitely wasn't something we expected given some of our limitations for virtual architectures on BN. 

Also, as an FYI -- we're tracking new releases automatically no need to let us know when it's updated. I just kicked off the update, if it got 1.0.0 instead of 1.0.1 I'll make sure to run it again.
